internal/useragent/useragent.go

```diff
@@ -39,6 +39,11 @@ func GRPCDialOption(api string) grpc.DialOption {
 	return grpc.WithUserAgent(userAgentString(api))
 }
 
+// AzureUserAgentPrefix returns a prefix that is used to set Azure SDK User-Agent to help with diagnostics.
+func AzureUserAgentPrefix() string {
+	return fmt.Sprintf("%s/%s", prefix, version)
```

Comment:
For GCP we included the API name (see below) so you can distinguish between uses other than `blob` once they exist, WDYT?

Reply:
Let me confirm with the PG; it doesn't hurt to include it. If so, can I make userAgentString public to avoid adding another helper function?

Reply:
Sure.

Reply:
Actually, it doesn't make sense exporting func userAgentString; I just amended my original helper.